raiden/channel.py

```diff
@@ -978,7 +1004,8 @@ def register_transfer_from_to(self, transfer, from_state, to_state):  # noqa pyl
                         transfer.recipient,
                         transfer.sender,  # 'initiator' is sender here
                         transfer.transferred_amount,
-                        None  # no hashlock in DirectTransfer
+                        None,  # no hashlock in DirectTransfer
+                        transfer.identifier
```

Comment:
This codepath seems to be hit when a Raiden `TransferReceived` event hits. The callback seems to be calling [this](https://github.com/raiden-network/raiden/pull/388/files/b8262bb56652b8947e6cc405e09550b2240c6bfe#diff-72b8041548bf416a299ac073c02e24e5R729) function which in turn populates the [`Transfer Received`](https://github.com/raiden-network/raiden/pull/388/files/b8262bb56652b8947e6cc405e09550b2240c6bfe#diff-a3273867481dfe86633875e7f489b66fR36) Raiden API object which takes an identifier as an argument.